src/azure-cli/azure/cli/command_modules/apim/_help.py

```diff
@@ -252,6 +257,76 @@
         az apim api export -g MyResourceGroup --service-name MyApim --api-id MyApi --export-format OpenApiJson --file-path path
 """
 
+helps['apim policy set'] = """
+type: command
+short-summary: Create or update policy for the given Scope
```

Comment:
```suggestion
short-summary: Create or update policy for the given scope
```

Reply:
There are many similar issues that need to be modified together below, so I won't repeat them